releng/tag-release.sh

```diff
@@ -108,19 +108,28 @@ if echo "${release}" | grep -q "[A-Za-z]"; then
   prerelease="--prerelease"
 fi
 
+[ -d "releng/assets/${release}" ] || mkdir -p "releng/assets/${release}"
+
+asset_dir="$( realpath -e "releng/assets/${release}" )"
+asset_files=()
+
 # Create binary assets
 if ! releng/make-binary.sh "${release}"; then
   error "ERROR: unable to make release assets, exiting!"
 fi
 
+# Copy in any extra assets/files, relative to the project root
+# shellcheck disable=SC2043
+for extra in bin/zbm-efi-kcl ; do
+  [ -f "${extra}" ] || error "ERROR: missing ${extra}"
+  cp "${extra}" "${asset_dir}/"
```

Comment:
```suggestion
  cp "${extra}" "${asset_dir}/"
  assets+=( "${extra##*/}" )
```

Reply:
Ahh yes. Missed that. I only checked that the signing script just globs the releng directory and signs everything in it.